client/src/reducers/annoMatrix.ts

```diff
@@ -4,11 +4,14 @@ Reducer for the annoMatrix
 
 import { AnyAction } from "redux";
 import AnnoMatrix from "../annoMatrix/annoMatrix";
+import { AnnoMatrixClipView } from "../annoMatrix/views";
+
+export type AnnoMatrixState = AnnoMatrix | AnnoMatrixClipView | undefined;
 
 const AnnoMatrixReducer = (
-  state: AnnoMatrix | null = null,
+  state: AnnoMatrixState = undefined,
   action: AnyAction
-) => {
+): AnnoMatrixState => {
```

Comment:
These changes lead to a lot of downstream issues with typing since the state can be undefined

Reply:
Good question, I think it's just for the app initialization, pre-data fetching where there is no annomatrix

Reply:
I imagine there are some typing difficulties somewhere as a result, but the effect size is probably smaller than the current alternative.